code/style.css

```diff
@@ -1,3 +1,185 @@
-body {
-  background: #FFECE9;
+ body {
+  color: #2C3844;
+  min-width: 325px;
+  font-family: 'Oswald', sans-serif;
+  background-color: #BBBFCA;
+}
+
+a {
+  text-decoration: none;
+  display: inline-block;
+	padding: 0.3em;
+	transition: .3s;
+	transform: scale(1);
+  color: #2C3844;
+}
+
+h2 {
+  text-align: center;
+  font-size: 30px;
+  margin-top: 0;
+}
+
+.big-container a:hover {
```

Comment:
I loved this hover effect - it looks amazing! 
And also the other styling of the page is supernice - you did a great job with it : D!